nes/include/nes/nes.h

```diff
@@ -45,6 +48,9 @@ class Nes {
     n_e_s::core::PpuRegisters &ppu_registers();
     const n_e_s::core::PpuRegisters &ppu_registers() const;
 
+    n_e_s::core::INesController &controller1() const;
+    n_e_s::core::INesController &controller2() const;
```

Comment:
Can you make both const, and non-const versions of these?

Reply:
Fixed!